.devcontainer/post_create.sh

```diff
@@ -0,0 +1,4 @@
+pushd site && bundle install && popd
+#yarn link
```

Comment:
Remove commented out code or run it. This isn't discoverable right now.

Reply:
removed it for now